variables.tf

```diff
@@ -0,0 +1,89 @@
+################################################################################
+# mongodbatlas_project
+################################################################################
+
+variable "org_id" {
+  type = string
+}
+variable "project_name" {
+  type    = string
+  default = "Default project name"
+}
+
+################################################################################
+# mongodbatlas_project_ip_access_list
+################################################################################
+
+variable "ip_addresses" {
+  type    = list(string)
+  default = []
+}
+
+variable "cidr_blocks" {
+  type    = list(string)
+  default = []
+}
+
+################################################################################
+# mongodbatlas_database_user
+################################################################################
+
+variable "database_users" {
+  type = list(object({
+    username = string
+    password = string
+    roles = optional(list(object({
+      role     = string
+      database = string
+    })), [{ role = "atlasAdmin", database = "admin" }])
+    scopes = optional(list(object({
+      name = string
+      type = string
+    })), [])
+  }))
+  default = []
+}
+
+################################################################################
+# mongodbatlas_advanced_cluster
+################################################################################
+
+variable "cluster_name" {
+  type    = string
+  default = "Cluster0"
+}
+
+variable "provider_name" {
+  type    = string
+  default = "AWS"
+}
+
+variable "backing_provider_name" {
```

Comment:
maybe we can add a description to the variables?

Reply:
The descriptions of inputs/outputs will be done in a different ticket / PR